x/erc20/keeper/migrations.go

```diff
@@ -21,5 +23,5 @@ func NewMigrator(keeper Keeper) Migrator {
 
 // Migrate1to2 migrates from consensus version 1 to 2.
 func (m Migrator) Migrate1to2(ctx sdk.Context) error {
-	return nil
```

Comment:
@fedekunze @khoslaventures not sure why the updateparams was not being called here?

Reply:
good catch